src/tools/updateLegacyPopulations.js

```diff
@@ -0,0 +1,103 @@
+import { Op } from 'sequelize';
+import { ActivityReport } from '../models';
+import { auditLogger } from '../logger';
+
+const newPopulations = {
+  'Infant/Toddlers': 'Infants and Toddlers (ages birth to 3)',
+  Preschool: 'Preschool (ages 3-5)',
+  'Affected by Homelessness': 'Children Experiencing Homelessness',
+};
+
+export default async function updateLegacyPopulations() {
+  auditLogger.info(`Updating legacy target population data...
+  
+  `);
+
+  const reports = await ActivityReport.findAll({
+    attributes: ['id', 'targetPopulations', 'imported'],
+    where: {
+      [Op.or]: [
+        {
+          targetPopulations: {
```

Comment:
Is this also updating non-legacy reports?

Reply:
Should be both, yeah